backend/src/v2/driver/driver.go

```diff
@@ -292,6 +292,25 @@ func Container(ctx context.Context, opts Options, mlmd *metadata.Client, cacheCl
 		}
 		execution.Condition = &willTrigger
 	}
+
+	// When the container image is a dummy image, there is no launcher for this
```

Comment:
This comment and conditional check are just moved up (and wrapped) from a few lines down. I also renamed the boolean to be more intuitive.